examples/crowdfunding/test/crowdfundingTest.js

```diff
@@ -0,0 +1,252 @@
+import {
+  addressToBytes,
+  getProgram,
+  intToBigEndian,
+  SignType,
+  stringToBytes,
+  TransactionType
+} from '@algorand-builder/algob';
+import { Runtime, StoreAccountImpl } from '@algorand-builder/algorand-js/build/src/index';
+import { assert } from 'chai';
+
+import { getAcc } from './common';
+
+const initialDonorBalance = 60000000;
+const initialCreatorBalance = 10000;
+const goal = 7000000;
+
+describe('Crowdfunding Tests', function () {
+  let creatorAccount = new StoreAccountImpl(initialCreatorBalance);
+  let escrowAccount = new StoreAccountImpl(0);
+  let donorAccount = new StoreAccountImpl(initialDonorBalance);
+
+  let runtime;
+  let program;
+  let flags;
+  let applicationId;
+  this.beforeAll(async function () {
+    runtime = new Runtime([creatorAccount, escrowAccount, donorAccount]);
+    program = getProgram('crowdFundApproval.teal');
+
+    flags = {
+      sender: creatorAccount.account,
+      localInts: 1,
+      localBytes: 0,
+      globalInts: 5,
+      globalBytes: 3
+    };
+  });
+
+  // update account state
+  function syncAccounts () {
+    creatorAccount = getAcc(runtime, creatorAccount);
+    escrowAccount = getAcc(runtime, escrowAccount);
+    donorAccount = getAcc(runtime, donorAccount);
+  }
+
+  it('should create crowdfunding application with correct global states', async () => {
+    const creationFlags = Object.assign({}, flags);
+
+    // Get begin date to pass in
+    const beginDate = new Date();
+    beginDate.setSeconds(beginDate.getSeconds() + 2);
+
+    // Get end date to pass in
+    const endDate = new Date();
+    endDate.setSeconds(endDate.getSeconds() + 12000);
+
+    // Get fund close date to pass in
+    const fundCloseDate = new Date();
+    fundCloseDate.setSeconds(fundCloseDate.getSeconds() + 120000);
+
+    const appArgs = [
+      intToBigEndian(beginDate.getTime()),
+      intToBigEndian(endDate.getTime()),
+      intToBigEndian(goal),
+      addressToBytes(creatorAccount.address),
+      intToBigEndian(fundCloseDate.getTime())
+    ];
+
+    const appId = await runtime.addApp({ ...creationFlags, appArgs: appArgs }, {}, program);
+    applicationId = appId;
+    const creatorPk = addressToBytes(creatorAccount.address);
+
+    // verify global state
+    assert.isDefined(appId);
+    assert.deepEqual(runtime.getGlobalState(appId, stringToBytes('Creator')), creatorPk);
+    assert.deepEqual(runtime.getGlobalState(appId, stringToBytes('StartDate')), BigInt(beginDate.getTime()));
+    assert.deepEqual(runtime.getGlobalState(appId, stringToBytes('EndDate')), BigInt(endDate.getTime()));
+    assert.deepEqual(runtime.getGlobalState(appId, stringToBytes('Goal')), BigInt(7000000));
```

Comment:
Please use `7000000n` instead of `BigInt('7000000')`

Reply:
it's not working with `linter`. i tried changing some rules but that didn't work. Please let me know if you find it

Reply:
I think we need to update the `ecmaVersion` as well. I got it working in my PR. [https://github.com/scale-it/algorand-builder/pull/184/commits/5d28f213a2d1a9ee009f42124593cd107d0faffc](https://github.com/scale-it/algorand-builder/pull/184/commits/5d28f213a2d1a9ee009f42124593cd107d0faffc)

Reply:
ok, thanks